src/utils/util_conv.cc

```diff
@@ -138,6 +138,134 @@ drake::lcmt_iiwa_status ConvertToLcmStatus(
+  std::copy(robot_status.ddelbow_c.begin(), robot_status.ddelbow_c.end(),
+            std::begin(status_msg.ddelbow_c));
+  std::copy(robot_status.tau_J.begin(), robot_status.tau_J.end(),
+            std::begin(status_msg.tau_J));
+  std::copy(robot_status.tau_J_d.begin(), robot_status.tau_J_d.end(),
+            std::begin(status_msg.tau_J_d));
+  std::copy(robot_status.dtau_J.begin(), robot_status.dtau_J.end(),
+            std::begin(status_msg.dtau_J));
+  std::copy(robot_status.q.begin(), robot_status.q.end(),
+            std::begin(status_msg.q));
+  std::copy(robot_status.q_d.begin(), robot_status.q_d.end(),
+            std::begin(status_msg.q_d));
+  std::copy(robot_status.dq.begin(), robot_status.dq.end(),
+            std::begin(status_msg.dq));
+  std::copy(robot_status.dq_d.begin(), robot_status.dq_d.end(),
+            std::begin(status_msg.dq_d));
+  std::copy(robot_status.ddq_d.begin(), robot_status.ddq_d.end(),
+            std::begin(status_msg.ddq_d));
+  std::copy(robot_status.joint_contact.begin(),
+            robot_status.joint_contact.end(),
+            std::begin(status_msg.joint_contact));
+  std::copy(robot_status.cartesian_contact.begin(),
+            robot_status.cartesian_contact.end(),
+            std::begin(status_msg.cartesian_contact));
+  std::copy(robot_status.joint_collision.begin(),
+            robot_status.joint_collision.end(),
+            std::begin(status_msg.joint_collision));
+  std::copy(robot_status.cartesian_collision.begin(),
+            robot_status.cartesian_collision.end(),
+            std::begin(status_msg.cartesian_collision));
+  std::copy(robot_status.tau_ext_hat_filtered.begin(),
+            robot_status.tau_ext_hat_filtered.end(),
+            std::begin(status_msg.tau_ext_hat_filtered));
+  std::copy(robot_status.O_F_ext_hat_K.begin(),
+            robot_status.O_F_ext_hat_K.end(),
+            std::begin(status_msg.O_F_ext_hat_K));
+  std::copy(robot_status.K_F_ext_hat_K.begin(),
+            robot_status.K_F_ext_hat_K.end(),
+            std::begin(status_msg.K_F_ext_hat_K));
+  std::copy(robot_status.O_dP_EE_d.begin(), robot_status.O_dP_EE_d.end(),
+            std::begin(status_msg.O_dP_EE_d));
+  std::copy(robot_status.O_T_EE_c.begin(), robot_status.O_T_EE_c.end(),
+            std::begin(status_msg.O_T_EE_c));
+  std::copy(robot_status.O_dP_EE_c.begin(), robot_status.O_dP_EE_c.end(),
+            std::begin(status_msg.O_dP_EE_c));
+  std::copy(robot_status.O_ddP_EE_c.begin(), robot_status.O_ddP_EE_c.end(),
+            std::begin(status_msg.O_ddP_EE_c));
+  std::copy(robot_status.theta.begin(), robot_status.theta.end(),
+            std::begin(status_msg.theta));
+  std::copy(robot_status.dtheta.begin(), robot_status.dtheta.end(),
+            std::begin(status_msg.dtheta));
+
+  status_msg.control_command_success_rate =
+      status_msg.control_command_success_rate;
+
+  status_msg.robot_mode = static_cast<int16_t>(robot_status.robot_mode);
```

Comment:
is this a narrowing cast? what's the consequences? add comments

Reply:
then why is the static_cast needed?

Reply:
It casts from an `enum class` to `int16_t`

Reply:
I suppose that is a narrowing cast in a sense :thinking:

Reply:
yeah it is. the default enum type is int (32). if your enum size is small, consider specifying `int64_t` in the enum class definition so no narrowing cast is done here

Reply:
But there are like 7 robot modes. Wouldn't `int64_t` be overkill?

Reply:
sorry typo, meant `int16_t`

Reply:
Oh, that enum is in `libfranka`

Reply:
nvm then, some comments would have been helpful

Reply:
I'll add some comments